app/controllers/IndexController.scala

```diff
@@ -16,107 +16,69 @@
-        } else {
-          result
-            .removingFromSession(allKeysExcludingAgentVRN: _*)
-        }
-      } else {
-      penaltiesServiceV2.getPenaltyDataFromEnrolmentKey(EnrolmentKeys.constructMTDVATEnrolmentKey(request.vrn)).map {
-        _.fold(
-          errors => {
-            logger.error(s"[OtherReasonController][getPenaltyDetailsFromNewAPI] - Received error with status ${errors.status} and body ${errors.body} rendering ISE.")
-            errorHandler.showInternalServerError
-          }, penaltyData => {
-            val contentToDisplayAboveCards = pageHelperv2.getContentBasedOnPointsFromModel(penaltyData)
-            val contentLPPToDisplayAboveCards = pageHelperv2.getContentBasedOnLatePaymentPenaltiesFromModel(penaltyData)
-            val whatYouOweBreakdown = pageHelperv2.getWhatYouOweBreakdown(penaltyData)
-            val lspSummaryCards = cardHelper2.populateLateSubmissionPenaltyCard(penaltyData.lateSubmissionPenalty.map(_.details).getOrElse(Seq.empty),
-              penaltyData.lateSubmissionPenalty.map(_.summary.regimeThreshold).getOrElse(0),
-              penaltyData.lateSubmissionPenalty.map(_.summary.activePenaltyPoints).getOrElse(0))
-            val lppSummaryCards = cardHelper2.populateLatePaymentPenaltyCard(penaltyData.latePaymentPenalty.map(_.details))
-            val isAnyUnpaidLSPAndNotSubmittedReturn = penaltiesServiceV2.isAnyLSPUnpaidAndSubmissionIsDue(penaltyData.lateSubmissionPenalty.map(_.details).getOrElse(Seq.empty))
-            val isAnyUnpaidLSP = penaltiesServiceV2.isAnyLSPUnpaid(penaltyData.lateSubmissionPenalty.map(_.details).getOrElse(Seq.empty))
-            val latestLSPCreation = penaltiesServiceV2.getLatestLSPCreationDate(penaltyData.lateSubmissionPenalty.map(_.details).getOrElse(Seq.empty))
-            lazy val result = Ok(view2(contentToDisplayAboveCards,
-              contentLPPToDisplayAboveCards,
-              lspSummaryCards,
-              lppSummaryCards,
-              currencyFormatAsNonHTMLString(penaltyData.totalisations.flatMap(_.LSPTotalValue).getOrElse(0)),
-              isAnyUnpaidLSP,
-              isAnyUnpaidLSPAndNotSubmittedReturn,
-              whatYouOweBreakdown))
-            if (latestLSPCreation.isDefined) {
-              result
-                .removingFromSession(allKeysExcludingAgentVRN: _*)
-                .addingToSession(latestLSPCreationDate -> latestLSPCreation.get.toString,
-                  pointsThreshold -> penaltyData.lateSubmissionPenalty.map(_.summary.regimeThreshold).getOrElse(0).toString)
-            } else {
-              result
-                .removingFromSession(allKeysExcludingAgentVRN: _*)
-            }
+    penaltiesServiceV2.getPenaltyDataFromEnrolmentKey(EnrolmentKeys.constructMTDVATEnrolmentKey(request.vrn)).map {
+      _.fold(
+        errors => {
+          logger.error(s"[OtherReasonController][getPenaltyDetails] - Received error with status ${errors.status} and body ${errors.body} rendering ISE.")
```

Comment:
The log doesn't match the method name
```suggestion
          logger.error(s"[IndexController][onPageLoad] - Received error with status ${errors.status} and body ${errors.body} rendering ISE.")
```